From 9234ff4eac87a5b635a6f19fa12e081d628dcf61 Mon Sep 17 00:00:00 2001
From: Stef Walter <stefw@collabora.co.uk>
Date: Wed, 28 Sep 2011 14:01:26 +0200
Subject: [PATCH 2/8] Skip analysis of params that have been (skip)'d

Otherwise we get warnings for params that have been skipped

https://bugzilla.gnome.org/show_bug.cgi?id=660352
---
 giscanner/introspectablepass.py |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/giscanner/introspectablepass.py b/giscanner/introspectablepass.py
index 95d54b0..77a0e4f 100644
--- a/giscanner/introspectablepass.py
+++ b/giscanner/introspectablepass.py
@@ -74,6 +74,9 @@ class IntrospectablePass(object):
         else:
             target = None
 
+        if node.skip:
+            return
+
         if not node.type.resolved:
             self._parameter_warning(parent, node,
 "Unresolved type: %r" % (node.type.unresolved_string, ))
-- 
1.7.7

